.travis.yml

```diff
@@ -28,6 +28,6 @@ before_install:
 - echo "password = $TX_PASSWD" >> ~/.transifexrc
 script:
 - yarn test
-- yarn run build
+- yarn build:browser
```

Comment:
I type `yarn <script>` everytime and I did not face yet an error. But it is not documented explicitly in the documentation.